tests/masonry/builders/test_wheel.py

```diff
@@ -451,3 +451,16 @@ def test_build_py_only_included() -> None:
 
     with zipfile.ZipFile(str(whl)) as z:
         assert "build_with_build_py_only/generated/file.py" in z.namelist()
+
+
+def test_generated_script_file() -> None:
+    """Tests that a file that is generated by build.py can be used as script."""
+    root = fixtures_dir / "generated_script_file"
```

Comment:
After executing the test once, the generated file is in the source tree and the test succeeds even without the change. Can you please use the `tmp_path` fixture, `shutil.copytree()` the project and work on the temporary path?

Reply:
@radoering thanks for the suggestion! Done in a380348.